src/uu/cksum/Cargo.toml

```
@@ -18,6 +18,7 @@ path = "src/cksum.rs"
 clap = { workspace = true }
 uucore = { workspace = true, features = ["encoding", "sum"] }
```

Running the `cksum` tests with `cargo test --features "cksum" --no-default-features` fails without the `checksum` feature.
```suggestion
uucore = { workspace = true, features = ["checksum", "encoding", "sum"] }
```